.github/checkgroup.yml

```diff
@@ -141,18 +141,15 @@ subprojects:
       - "setup.py"
       - ".actions/**"
     checks:
-      - "build-cuda (3.8, 1.9, 11.1.1)"
       - "build-cuda (3.9, 1.10, 11.3.1)"
       - "build-cuda (3.9, 1.11, 11.3.1)"
       - "build-cuda (3.9, 1.12, 11.6.1)"
-      - "build-cuda (3.9, 1.9, 11.1.1)"
       - "build-hpu (1.5.0, 1.11.0)"
       - "build-ipu (3.9, 1.9)"
       - "build-NGC"
       - "build-pl (3.9, 1.10, 11.3.1)"
       - "build-pl (3.9, 1.11, 11.3.1)"
       - "build-pl (3.9, 1.12, 11.6.1)"
-      - "build-pl (3.9, 1.9, 11.1.1)"
```

Comment:
We could keep this as it is based on the existing image; just install PL inside...
anyway, we can drop it :otter: